src/components/shared/forms/styles/pillContainer.scss

```diff
@@ -0,0 +1,4 @@
+.pill-container{
```

Comment:
Add `// Copyright (c) Microsoft. All rights reserved.` at the top of the file.

Reply:
Added.